staging/src/k8s.io/apiserver/pkg/server/storage/storage_factory_test.go

```diff
@@ -88,6 +90,7 @@ func (n *fakeNegotiater) DecoderToVersion(serializer runtime.Decoder, gv runtime
 }
 
 func TestConfigurableStorageFactory(t *testing.T) {
+	utilfeature.DefaultFeatureGate.Add(features.DefaultKubernetesFeatureGates)
```

Comment:
why?

Reply:
Same as above (and of course all other occurences of this).